CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ### Major
 
-- Link: design changes (part I: inline/underline) (#2073)
+- Link: design changes (part I inline/underline) (#2073)
```

Comment:
This is typically a generated file — is this change intentional?

Reply:
Yes, to prevent
![Screen Shot 2022-05-02 at 1 05 51 PM](https://user-images.githubusercontent.com/10593890/166292360-b122cd01-da1f-4f86-90a9-c404b922fdbf.png)